default_kms_key_arn.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python2
+import sys
+import json
+import boto3
+
+if __name__ == '__main__':
```

Comment:
I think this if statement is unnecessary in this type of script because we're not going to be importing this from another module.